dac/backend/src/main/java/com/dremio/dac/support/BasicSupportService.java

```diff
@@ -455,9 +518,19 @@ public void start() throws Exception {
     supportPath = new File(supportPathURI).toPath();
   }
 
+  /**
+   * Downloads a support request as a zip file.  The support request file will contain the request's metadata,
+   * such as its submission ID and a summary of the Job executed.
+   *
+   * @param request the request to have its metadata downloaded
+   * @return an object with the downloaded file's metadata
+   * @throws UserNotFoundException If it couldn't find the user's profile
+   * @throws IOException           If any exception occurs while trying to create the downloaded file
+   * @throws JobNotFoundException  If it couldn't find the Job for the request
```

Comment:
Use the `<p>` tag to separate paragraphs and align the return with the param.

Reply:
done